core/systems/account/people_view.ex

```diff
@@ -74,159 +130,176 @@ defmodule Systems.Account.PeopleView do
     }
   end
 
-  @impl true
-  def handle_event(
-        "search_query",
-        %{query_string: query_string, source: %{name: :search_bar}},
-        socket
-      ) do
-    {
-      :noreply,
-      socket
-      |> assign(query_string: query_string)
-      |> update_user()
-      |> update_message()
-    }
-  end
+  # ------------------------------------------------------------------
+  # Private Helpers
+  # ------------------------------------------------------------------
 
-  @impl true
-  def handle_event(
-        "add",
-        %{"item" => user_id},
-        %{assigns: %{users: users, people: people}} = socket
-      ) do
-    user_id = String.to_integer(user_id)
+  defp assign_defaults(socket, users, people, title, current_user) do
```

Comment:
Do you like the approach of having defp at the nd of the file and therefor also after the render function? Just curious.

Reply:
I like to have my private methods separated from my public methods (funny that i keep using the term "method" when function should be used) to have clear "plateau's" of logic, if you get what I mean. It clearly separates the parts that the outside world can interact with from the internal logic of the module/class. Usually I use this kind of structure for myself to keep it more clear, but I also think that the render function should be the last function in a component, so I will change that. Please let me know what you think of this so we can get to a more standardized approach.